src/test/resources/config.json

```diff
@@ -10,16 +10,19 @@
   "gridConfig": {
     "url": "http://localhost:4444/wd/hub"
   },
-  "browserPreferences" : {
+  "browserPreferences": {
     "firefox": {
       "browser.download.defaultFolder": "downloads/reports",
       "pdfjs.disabled": true,
       "browser.download.folderList": 2
     }
   },
-  "exceptionsToHandleOnTolerantActions": [
-    "StaleElementReferenceException",
-    "ElementClickInterceptedException",
-    "ElementNotInteractableException"
-  ]
+  "tolerantActionExceptions": {
+    "tolerantActionWaitTimeoutInSeconds": 5,
+    "exceptionsToHandleOnTolerantActions": [
```

Comment:
We can just call this `exceptionsToHandle` i think because we're in an encapsulating object that gives context as to what it is.

Reply:
changed the property name from "tolerantActionWaitTimeoutInSeconds" to "exceptionsToHandle" as mentioned above